### ISSUE TITLE
Compile fails with gcc 7.5 "sorry, unimplemented: non-trivial designated initializers not supported"

### ISSUE THREAD

**Issue by KottV** [sjaehn/BShapr#17]
Hi,

With the recent change it fails to build with old gcc and g++ (for example openSUSE Leap 15.2)
```[    2s] Build BShapr.lv2 DSP...Build BShapr.lv2 GUI...Build BShapr.lv2 (CV version) DSP...Build BShapr.lv2 (CV version) GUI...../../src/BShaprGUI.cpp:1766:60: sorry, unimplemented: non-trivial designated initializers not supported
[    2s]  static const LV2UI_Resize resize = {.ui_resize = callResize} ;
[    2s]                                                             ^
[    2s] ../../src/BShaprGUI.cpp:1766:60: sorry, unimplemented: non-trivial designated initializers not supported
[    2s]  static const LV2UI_Resize resize = {.ui_resize = callResize} ;
```
With gcc 10 all is fine. This affects all updated B* plugins.

**Comment by sjaehn** [comment]
Thanks to mention. I'm just busy with. gcc < 8 and clang < 10 don't fully support C99 designated initializers. Users of gcc < 6 also experience errors do to lacking support of enumerator attributes. Fix is on the way.

**Comment by sjaehn** [comment]
Hopefully fixed in fe348b008c393e3d4e6f56c3db7f71930c7a99ff . Also applied to all other B.Music plugins.

**Comment by KottV** [comment]
Great! It compiles now) Thank You.
